src/components/util/object.js

```diff
@@ -158,12 +158,20 @@ export function getNestedProperty(object, propertyName, suppressNotFoundError =
 
             if (Array.isArray(currentNode)){
                 var arrayVals = [];
-                for (var i = 0; i < currentNode.length; i++){
-                    arrayVals.push( findNestedValue(currentNode[i], fieldHierarchyLevels, level) );
+                for (var i = 0; i < currentNode.length; i++) {
+                    if (typeof currentNode[i][fieldHierarchyLevels[level]] !== 'undefined') {
+                        arrayVals.push(findNestedValue(currentNode[i], fieldHierarchyLevels, level));
+                    } else {
+                        arrayVals.push(null);
+                    }
+                }
+                if (!_.any(arrayVals, (val) => val !== null)) {
```

Comment:
Might be more clear to write `if (_.any(arrayVals, function(val){ return val === null; }))` (avoiding double-negative '!').

----

This function is meant to be used in many places (not just search result columns) so would double check that other places that use it aren't negatively affected (shouldn't be, though). One potential consideration is if array full of `null`s or mixed values including `null`s is an actual value we want to get -- which if there is such value, we probably _do_ want to grab it instead of receiving a single `null` or error. It might be better to filter such array outside the scope of this function if so.

Reply:
@alexkb0009 `if(!_.any(arrayVals, (val) => val !== null))` might be rewritten as `if(_.all(arrayVals, (val) => val === null))` instead of `if(_.any(arrayVals, function(val){ return val === null; }))`, since we want to ensure at least one item in array is not null to avoid `Error`.

Reply:
Re: One potential consideration is if array full of nulls or mixed values including nulls is an actual value we want to get

FYI, current implementation ignores the array altogether even only single item of the array is null.